ocaml/xapi/xapi_pool.ml

```diff
@@ -3409,10 +3410,12 @@ let sync_updates ~__context ~self ~force ~token ~token_id =
              sync ~__context ~self:x ~token ~token_id ;
              create_pool_repository ~__context ~self:x
          ) ;
+      Db.Pool.set_last_update_sync ~__context ~self:pool ~value:(Date.now ()) ;
```

Comment:
The `set_available_updates` may fail. I think it's better to set the `last_update_sync` after `set_available_updates`.